### DISCUSSION TITLE
Styling markdown ?

### DISCUSSION THREAD

**Discussion by Altho** [ssssota/svelte-exmarkdown#21] [Q&A] [answered]
I'm using exMarkdown inside a component

`<Page>
    {data.post.title}
    <Markdown  {md} }/>
</Page>`

I feel like this is a pretty stupid question but I can't style the rendered html. No matter what I do the styling is not taking into account

Am I missing something here ?

Thanks.

**Comment by ssssota** [comment, marked as answer]
Thank you for asking!

You can like this: https://stackblitz.com/edit/svelte-exmarkdown-styling?file=src%2FApp.svelte

**Comment by Altho** [comment]
Awesome. Thanks you !
